packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1658,6 +1658,8 @@
   "Report_this_message_question_mark": "Report this message?",
   "Request": "Send Request",
   "Request_to_join_room": "You are not a member of this #<strong>__roomName__</strong> room. Would you like to send request to join",
```

Comment:
More grammar, please
You are not a member of this #<strong>__roomName__</strong> room.  Would you like to send *a* request for joining it?

de: Du bist bisher nicht Mitglied des Kanals. Möchtest Du fragen, ob Du ihm beitreten darfst?

Reply:
Translation Maintained.